scripts/kind-build-test.sh

```diff
@@ -97,8 +97,8 @@ Environment variables:
                             Default: aws-controllers-k8s:$AWS_SERVICE-$VERSION
   TMP_DIR                   Cluster context directory, if operating on an existing cluster
                             Default: $ROOT_DIR/build/tmp-$CLUSTER_NAME
-  K8S_VERSION               Kubernetes Version [1.14, 1.15, 1.16, 1.17, and 1.18]
-                            Default: 1.16
+  K8S_VERSION               Kubernetes Version [1.17, 1.18, 1.19, 1.20, 1.21, 1.22, 1.23 and 1.24]
```

Comment:
EKS no longer supports 1.17 or 1.18. I'd be fine with moving to 1.19+ now. No point going back to those versions.